core/src/main/kotlin/keiyoushi/utils/Preferences.kt

```diff
@@ -25,4 +25,8 @@ inline fun HttpSource.getPreferencesLazy(
  */
 @Suppress("NOTHING_TO_INLINE")
 inline fun getPreferences(sourceId: Long): SharedPreferences =
-    Injekt.get<Application>().getSharedPreferences("source_$sourceId", 0x0000)
+    getApplication().getSharedPreferences("source_$sourceId", 0x0000)
+
+// This ensures the type token isn't duplicated at each call site.
+@PublishedApi
+internal fun getApplication(): Application = Injekt.get()
```

Comment:
This was discovered in #10020. Even though this line was in an `inline` function, it still drops a type token here in this file, which is hard to optimize out due to how R8 rules are written. So I guess it's best to not duplicate it in the extension code.